package.json

```diff
@@ -1,14 +1,14 @@
 {
   "name": "modal-form",
-  "version": "2.9.1",
+  "version": "2.10.0",
   "devDependencies": {},
   "peerDependencies": {
-    "react": "^16.2.0",
-    "react-dom": "^16.2.0"
+    "react": "17.x",
+    "react-dom": "^17.x"
```

Comment:
The peer dependencies should include React 16 and React 17.  See Markdownz for an example.